app/src/main/kotlin/com/wire/android/ui/home/whatsnew/WhatsNewScreen.kt

```diff
@@ -74,13 +75,15 @@ fun WhatsNewScreenContent(
         state = lazyListState,
         modifier = modifier.fillMaxSize()
     ) {
-        folderWithElements(
-            items = buildList {
-                add(WhatsNewItem.WelcomeToNewAndroidApp)
-            },
-            onItemClicked = onItemClicked,
-            isLoading = false,
-        )
+        if (BuildConfig.SHOULD_DISPLAY_RELEASE_NOTES) {
```

Comment:
question: this is also the task to change the all notes link, will this be in a following PR?

Reply:
Yes did the change here https://github.com/wireapp/android-customization/pull/30